app/components/assets/actions-dropdown.tsx

```diff
@@ -15,12 +22,16 @@ interface Props {
     mainImage: Asset["mainImage"];
     status: Asset["status"];
   };
+  isQr: boolean;
 }
 
-export const ActionsDopdown = ({ asset }: Props) => {
+const ActionsDopdown = ({ asset, isQr }: Props) => {
   const assetIsAvailable = asset.status === "AVAILABLE";
+  const defaultOpen = window.innerWidth <= 640 && isQr;
```

Comment:
This causes an error. What I would suggest we do is use state for this with default value of false. Then you need an effect that runs, checks if `window` exists and only then set it to the correct value.
![Screenshot 2023-08-21 at 11 11 25](https://github.com/Shelf-nu/shelf.nu/assets/7840007/255693a1-c1c2-45f2-b047-c532c555f1c9)